azure-pipelines.yml

```diff
@@ -109,10 +109,6 @@ jobs:
         gcc8:
           containerImage: gcc:8
           cli11.std: 17
-        gcc4.8:
-          containerImage: gcc:4.8
```

Comment:
Is there a reason this can't just change to helics/buildenv:gcc4-8-builder ? Does `.ci/azure-cmake.yml` break?

Reply:
My guess would be no, but CMake is already installed so just seemed a waste of time, I will put it back to make sure.

Reply:
I put it back in the Docker section,  installing CMake takes about 30 seconds on the CI test, so we can reduce the build time slightly by leaving it in its own section, or just leave the extra CMake install in place.

Reply:
We've got enough jobs where I'm not too worried about it. I'd rather like to move to GHA eventually & then we can probably use the CMake action. Keeping this simple makes it easier to move. ;)